zapcore/write_syncer_bench_test.go

```diff
@@ -53,4 +56,51 @@ func BenchmarkMultiWriteSyncer(b *testing.B) {
 			}
 		})
 	})
+	b.Run("4 discarder with buffer", func(b *testing.B) {
+		w, close := Buffer(NewMultiWriteSyncer(
+			&ztest.Discarder{},
+			&ztest.Discarder{},
+			&ztest.Discarder{},
+			&ztest.Discarder{},
+		), 0, 0)
+		defer close()
+		b.ResetTimer()
+		b.RunParallel(func(pb *testing.PB) {
+			for pb.Next() {
+				w.Write([]byte("foobarbazbabble"))
+			}
+		})
+	})
+}
+
+func BenchmarkWriteSyncer(b *testing.B) {
+	b.Run("write file with no buffer", func(b *testing.B) {
+		file, err := ioutil.TempFile(".", "*")
```

Comment:
nit: just pass `""` as the directory to use the default, and pass in `"log"` as the file argument (it'll automatically have values appended, no need for `*`)

Reply:
alright, fixed